libpod/runtime_cstorage.go

```diff
@@ -112,12 +112,14 @@ func (r *Runtime) removeStorageContainer(idOrName string, force bool) error {
 			return errors.Wrapf(define.ErrCtrStateInvalid, "container %q is mounted and cannot be removed without using force", idOrName)
 		}
 	} else if _, err := r.store.Unmount(ctr.ID, true); err != nil {
-		if errors.Cause(err) == storage.ErrContainerUnknown {
+		if errors.Is(err, storage.ErrContainerUnknown) {
```

Comment:
No need for two ifs, throw the other one in here and or them together so we still get a good Infof.

Unless you think that `DeleteContainer` will still work if we get a `ErrLayerUnknown` but not an `ErrContainerUnknown`?

Reply:
Ah, reading back to the old conversation in the linked issue, that does appear to be the assumption...

Maybe we should just `logrus.Errorf()` any errors that come out of Unmount and unconditionally proceed to the `DeleteContainer()`?

Reply:
Ok how about now, we always try to delete, if we can not determine if the container is mounted or if we fail to unmount the container.

Reply:
Works for me